js/send-mail.js

```diff
@@ -122,7 +122,8 @@ $(function() {
 			},
 			type: 'POST',
 			success:function() {
-				Mail.UI.messageView.setMessageFlag(messageId, 'answered', true);
+				// Reload the message list to display the answered flag.
+				Mail.UI.messageView.loadMessages();
```

Comment:
The setMessageFlag didn't have a method for answered, and using Backbone functions get/set is inconsistent; sometimes TypeErrors are abound.